app/views/course/achievements/_form.html.slim

```diff
@@ -27,7 +27,7 @@
           th = t('course.condition.type.header')
           th = t('course.condition.description.header')
       tbody
-        - @achievement.conditions.includes(:conditional).each do |condition|
+        - @achievement.conditions.each do |condition|
```

Comment:
So it turns out this is needed, as `conditional` is used in the partials that are rendered in the block. I've reverted this in #425. Should I assign `@conditions` in the controller to eagerly load this?